pynamodb/pagination.py

```diff
@@ -0,0 +1,63 @@
+from pynamodb.constants import CAMEL_COUNT, ITEMS, LAST_EVALUATED_KEY
+
+
+class ResultIterator(object):
```

Comment:
Do you think everything in here is adequately tested by everything we're already doing, or are there any gaps? I _think_ pretty much all of this should already be covered but just want to ask the question.

Reply:
I believe so -- we have test cases in model that handle different combinations of limits and result page sizes